webserver/views/api/v1/core.py

```diff
@@ -193,7 +193,12 @@ def get_many_lowlevel():
 
     # TODO: This should use a bulk get method for speed
     for recording_id, offset in recordings:
-        recording_details.setdefault(recording_id, {})[offset] = db.data.load_low_level(recording_id, offset)
+        try:
+            recording_details.setdefault(recording_id, {})[offset] = db.data.load_low_level(recording_id, offset)
+        except NoDataFoundException:
+            errors = recording_details.setdefault("errors", {})
```

Comment:
I've been thinking about this `errors` key. I don't think it's necessary. My opinion now is that if a recording doesn't exist, we just skip it.